CONTRIBUTING.md

```diff
@@ -57,6 +57,22 @@ Note: `release/x.y` or `hotfix/x.y.z` will be merged back in `master` after a ne
 
 [git-flow]: http://nvie.com/posts/a-successful-git-branching-model/
 
+# Subtree'd library projects
+
+A number of library dependencies are managed as separate open source projects and are git-subtree'd into the WordPress Android app source tree. Use the following command to updated (pull latest) from their respective repos:
+
+        $ git subtree pull --prefix libs/library_name https://github.com/wordpress-mobile/WordPress-Library_Name-Android.git develop
```

Comment:
Better to add the `--squash` flag by default here, make the history cleaner in the main repo. We should also add the command line to push changes from the main repo to the subtree repo.